client-core/src/hdwallet/key_chain/chain_path.rs

```diff
@@ -0,0 +1,165 @@
+use crate::hdwallet::KeyIndex;
```

Comment:
if this file was ported from the other project, it'll be good to add the attribution header as in https://github.com/crypto-com/chain/pull/475/files#diff-3b1a2b5c81eaeed4652a670819839339R2